<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>campaign_and_advertisements</class>
 <widget class="QDialog" name="campaign_and_advertisements">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Campaign and advertisements</string>
  </property>
  <widget class="QLabel" name="phoneLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>52</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>13</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Campaign ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="areaLabel">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>208</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Budget</string>
   </property>
  </widget>
  <widget class="QLabel" name="typeLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>169</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Goal*</string>
   </property>
  </widget>
  <widget class="QLabel" name="emailLabel">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>90</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start date</string>
   </property>
  </widget>
  <widget class="QLabel" name="budgetLabel">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>247</y>
     <width>173</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Company name</string>
   </property>
  </widget>
  <widget class="QLabel" name="addressLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>End date</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="advertisementsTableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>721</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>520</y>
     <width>481</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previousButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>60</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
  <widget class="QToolButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>490</y>
     <width>36</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QToolButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>490</y>
     <width>55</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QToolButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>490</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Edite</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>17</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="endDateEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>136</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="companyComboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>256</y>
     <width>261</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>56</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="startDateEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>96</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="goalLineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>176</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="placeholderText">
    <string>Money</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="budgetSpinBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximum">
    <number>10000000</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
